src/vcpkg/commands.set-installed.cpp

```diff
@@ -60,24 +53,25 @@ namespace vcpkg
             args.github_workflow.has_value() && args.github_run_id.has_value())
         {
             Json::Object detector;
-            detector.insert("name", Json::Value::string("vcpkg"));
-            detector.insert("url", Json::Value::string("https://github.com/microsoft/vcpkg"));
-            detector.insert("version", Json::Value::string("1.0.0"));
+            detector.insert(JsonIdName, Json::Value::string(JsonIdVcpkg));
```

Comment:
`JsonIdVcpkg` in this case seems more like an "unrelated constant" like the URL and version below.